cmd/vic-machine/create/create.go

```diff
@@ -408,6 +408,12 @@ func (c *Create) Run(cli *cli.Context) error {
 		return err
 	}
 
+	if len(cli.Args()) > 0 {
```

Comment:
We can keep it simple and report the following - this makes it much easier for a user to tell what the complaint is about:
```
log.Errorf("Unknown argument: %s", cli.Args()[0])
```